spec/lib/vmdb/util_spec.rb

```diff
@@ -6,46 +6,46 @@
     end
 
     it "without a host" do
-      stub_server_configuration(:http_proxy => {})
```

Comment:
Please add a spec for testing the backward compatible settings. In case the customer did not start with a new settings file. Naming it like 'it "returns proxy for old settings"'